community/import-tool/src/test/java/org/neo4j/tooling/ImportToolTest.java

```diff
@@ -1176,6 +1182,30 @@ public void shouldBeEquivalentToUseRawAsciiOrCharacterAsQuoteConfiguration2() th
         }
     }
 
+    @Test
+    public void shouldRespectDbConfig() throws Exception
+    {
+        // GIVEN
+        int size = 10;
```

Comment:
can we use two different sizes here, just to make sure they land up individually correctly?